python/test/pipeline_test.py

```diff
@@ -450,10 +451,10 @@ def testStartCountdown(self):
   def testStartEta(self):
     """Tests starting a pipeline with an eta."""
     stage = NothingPipeline('one', 'two', three='red', four=1234)
-    eta = datetime.datetime.utcnow() + datetime.timedelta(seconds=30)
+    eta = datetime.datetime.utcnow().replace(tzinfo=pytz.UTC) + datetime.timedelta(seconds=30)
```

Comment:
This should not be necessary. According to the pipeline docs (and taskqueue - https://cloud.google.com/appengine/docs/python/taskqueue/tasks#eta_arg) eta could be provided with or without timezone information). What I don't understand is why are we passing utcnow() instead of now()?